src/plugins/data/public/ui/filter_bar/filter_editor/lib/filter_label.test.tsx

```diff
@@ -18,12 +18,10 @@
  */
 
 import React from 'react';
-import { FilterLabel } from './filter_label';
-import { render, cleanup } from '@testing-library/react/pure';
+import FilterLabel from './filter_label';
+import { render } from '@testing-library/react';
 import { phraseFilter } from '../../../../stubs';
 
-afterEach(cleanup);
```

Comment:
These `cleanups` should no longer be needed as explained in https://github.com/elastic/kibana/pull/78742